plugins/reporters/spdx/src/main/kotlin/SpdxDocumentReporter.kt

```diff
@@ -89,3 +104,25 @@ class SpdxDocumentReporter : Reporter {
         }
     }
 }
+
+private fun SpdxDocument.getLicenseRefExceptions(): Set<String> =
+    buildSet {
+        files.forEach { file ->
+            file.licenseInfoInFiles.forEach { license ->
```

Comment:
Should we maybe implement this logic rather in a `List<String>`- / `String`-serializer that we could easily annotate more fields than just `licenseInfoInFiles` with?

Reply:
Not sure if I get your point. I guess, it's fine to do that once it is needed.

Reply:
> Not sure if I get your point.

My point is to implement this in a way so that the check can easily be applied to other properties, like `SpdxPackage.licenseInfoFromFiles`, without too much hard-coding or code duplication.

Reply:
I've adjusted the code to also consider `licenseInfoFromFiles`, which shows that the approach almost as-is works without too much hard-coding or code duplication.